containers/arangodb38ubi.docker/Dockerfile

```diff
@@ -24,9 +24,8 @@ ADD ./LICENSE /licenses/LICENSE
 
 ENV GLIBCXX_FORCE_NEW=1
 
-RUN microdnf install gpg nodejs wget binutils && \
-    npm install -g foxx-cli && \
```

Comment:
Is Alan aware of this change?

Reply:
That is what git think happened. In reality 3.5 docker sources were deleted and 3.8 were made from devel. But it treated this as 3.5 renamed to 3.8 with changes. Therefore these changes are there for a long time and were approved by @pluma since 3.6.